### PR TITLE
New package: Liquit.Workspace.Agent42 version 4.2.3735.5913

### PR DESCRIPTION
**Pull request by rovabunw**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/203719)

### CODE REVIEW

File: manifests/l/Liquit/Workspace/Agent42/4.2.3735.5913/Liquit.Workspace.Agent42.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Architecture: x64
+  InstallerUrl: https://download.liquit.com/release/4.2/3735/Liquit-Universal-Agent-Win-4.2.3735.5913-x64.msi
+  InstallerSha256: B2E799C87E335D165257EC63AD180DD5C1D730878296ACB3A01D80FCF530910E
+  ProductCode: '{F0D42568-BD85-4A9F-A330-1C095773BEE7}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liquit Universal Agent for Windows (x64)
+    Publisher: Liquit Software B.V.
+    DisplayVersion: 4.2.3735.5913
+    ProductCode: '{F0D42568-BD85-4A9F-A330-1C095773BEE7}'
+    UpgradeCode: '{2B15E482-04E7-48A7-897A-12E5F877A2FF}'
```
Comment (mdanish-kh) [inline manifests/l/Liquit/Workspace/Agent42/4.2.3735.5913/Liquit.Workspace.Agent42.installer.yaml:18]:
```suggestion
  AppsAndFeaturesEntries:
  - DisplayName: Liquit Universal Agent for Windows (x64)
    UpgradeCode: '{2B15E482-04E7-48A7-897A-12E5F877A2FF}'
```

File: manifests/l/Liquit/Workspace/Agent42/4.2.3735.5913/Liquit.Workspace.Agent42.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Architecture: x86
+  InstallerUrl: https://download.liquit.com/release/4.2/3735/Liquit-Universal-Agent-Win-4.2.3735.5913-x86.msi
+  InstallerSha256: F6254B40068ABA6950CD4DBEC0F50579EB56C70881118F153088DAC8D39B9011
+  ProductCode: '{7BB1F40D-A8A0-4A66-A5D7-3AC0EE8A7651}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liquit Universal Agent for Windows (x86)
+    Publisher: Liquit Software B.V.
+    DisplayVersion: 4.2.3735.5913
+    ProductCode: '{7BB1F40D-A8A0-4A66-A5D7-3AC0EE8A7651}'
+    UpgradeCode: '{2B15E482-04E7-48A7-897A-12E5F877A2FF}'
```
Comment (mdanish-kh) [inline manifests/l/Liquit/Workspace/Agent42/4.2.3735.5913/Liquit.Workspace.Agent42.installer.yaml:28]:
```suggestion
  AppsAndFeaturesEntries:
  - DisplayName: Liquit Universal Agent for Windows (x86)
    UpgradeCode: '{2B15E482-04E7-48A7-897A-12E5F877A2FF}'
```

**Comment by stephengillie** [comment]
Verify:
image
The application seems to be tray-based, but appeared to have exited. So I wasn't able to have it display version data.

**Review by stephengillie — APPROVED** [review]

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
@stephengillie - is there a way your automation can check for the condition DisplayVersion == PackageVersion and not approve it in that case?

**Comment by stephengillie** [comment]
Hi @,

This manifest removes the DisplayVersion from the AppsAndFeaturesEntries, which is present in previous manifest versions. This entry should be added to this version, to maintain version matching, and prevent the 'upgrade always available' situation with this package.

(Automated response - build 933.)

**Pull request merged**